Assets/Tests/InputSystem/CoreTests_Actions.cs

```diff
@@ -10045,6 +10048,117 @@ public void Actions_RebindingCandidatesShouldBeSorted_IfAddingNewCandidate()
         }
     }
 
+    // Straight from demo project
```

Comment:
Which demo project? Bug repro project? If this is the case it might be good to include bug issue number in case we need to trace it at some point. I noticed its in the changelog, but maybe add a link to public issue tracker here?